code/script.js

```diff
@@ -303,41 +331,92 @@ const filterCharacters = (keep) => {
         `No, the person doesn't wear ${attribute}! Remove all that wears ${attribute}`
       )
     }
-  } else if (group === 'other') {
-    // Similar to the one above
+  } else if (category === 'other') {
+    if (keep) {
+      alert(
+        `Yes, the person is ${attribute}! Keep all persons who are ${attribute}`
+      )
+    } else {
+      alert(
+        `No, the person is not ${attribute}! Remove all persons who are not ${attribute}`
```

Comment:
A little observation while playing the game. Here when the secret person is "not smoking" the program removes the persons who are smoking ( as it is meant to do) but the message is saying that will remove the persons who are "NOT smoking". Maybe the word "not" in the alert should be removed so it matches the action of the program?